src/algebra/algebra/subalgebra.lean

```diff
@@ -648,6 +648,22 @@ instance : unique (subalgebra R R) :=
   end
   .. algebra.subalgebra.inhabited }
 
+/-- The map `S → T` when `S` is a subalgebra contained in the subalgebra `T`.  -/
```

Comment:
```suggestion
/-- The map `S → T` when `S` is a subalgebra contained in the subalgebra `T`.

This is the subalgebra version of `submodule.of_le`. -/
```
If we have the subring version it might be good to mention it here too